app/screens/Reusable/Acknowledgement/AcknowledgementAlongEdges/AcknowledgementAlongEdges.tsx

```diff
@@ -64,14 +38,19 @@ export const AcknowledgementAlongEdgesScreen: React.FC<IAcknowledgementAlongEdge
         }
         style={generalStyles.flex}>
         <SafeAreaView style={generalStyles.flex}>
-          {renderHeader()}
+          {useRenderQuestHeader(crossHeader ?? false)}
           <BottomButtonView
             buttonTitle={buttonTitle ?? t('buttons.next')}
             onSubmit={onSubmit}
             style={styles.container}>
-            <ExtendedText preset="title" style={styles.title}>
-              {useParseTextWithNickname(title)}
-            </ExtendedText>
+            {useParseTextWithNickname({
+              text: title,
+              textHasNickname: titleHasNickname ?? true,
+              preset: 'title',
+              style: styles.title,
+              // TODO: remove
+              nicknameStyle: { color: '#00dbc0' },
+            })}
```

Comment:
Hook should be here, I think you should use it before JSX and then put it as <YourVariableNameFromUppercasedLetter prop1 prop2 prop3 />